scripts/ingests/utils.py

```diff
@@ -65,6 +65,70 @@ def load_simpledb(db_file, RECREATE_DB=True):
     return db
 
 
+def find_in_simbad(sources, desig_prefix, source_id_index = None, verbose = False):
+    """
+    Function to extract source designations from SIMBAD
+
+    Parameters
+    ----------
+    sources
+        Sources names to search for in SIMBAD
+    desig_prefix
+        prefix to search for in list of identifiers
+    source_id_index
+    verbose
+
+    Returns
+    -------
+    Astropy table
+
+    """
+
+    n_sources = len(sources)
+
+    Simbad.reset_votable_fields()
+    Simbad.add_votable_fields('typed_id')  # keep search term in result table
+    Simbad.add_votable_fields('ids')  # add all SIMBAD identifiers as an output column
+
+    print("simbad query started")
+    result_table = Simbad.query_objects(sources['source'])
+    print("simbad query ended")
+
+    ind = result_table['SCRIPT_NUMBER_ID'] > 0  # find indexes which contain results
+    simbad_ids = result_table['TYPED_ID', 'IDS'][ind]
+
+    db_names = []
+    simbad_designations = []
+    if source_id_index is not None:
+        source_ids = []
+
+    for row in simbad_ids:
+        db_name = row['TYPED_ID']
+        ids = row['IDS'].split('|')
+        designation = [i for i in ids if desig_prefix in i]
+
+        if designation:
+            verboseprint(db_name, designation[0])
+            db_names.append(db_name)
+            simbad_designations.append(designation[0])
```

Comment:
Is there a possibility that more than one match happens and you want one other than the first one?

Reply:
I can make a warning if there’s multiple matches